ethpm_cli/parser.py

```diff
@@ -194,11 +206,5 @@ def list_action(args: argparse.Namespace) -> None:
 list_parser = ethpm_parser.add_parser(
     "list", help="List all installed packages in your ethPM directory."
 )
-list_parser.add_argument(
-    "--ethpm-dir",
```

Comment:
Curious about your choice to include this in all of these parsers rather than include it a single time in the main parser?

Reply:
It was mostly an aesthetic decision. Preferring something like
```
ethpm install <uri> --package-name xxx --version 1 --ethpm-dir /path
```
over
```
ethpm --ethpm-dir /path install <uri> --package-name xxx --version 1
```
I prefer putting the `ethpm-dir` arg at the end of the command, but maybe it's relatively insignificant? And I should just include the arg on the main parser, and simplify the code?

Reply:
That makes sense.  I think this is the API you're looking for to remove the repitition.

https://docs.python.org/3/library/argparse.html#parents

Reply:
Ok, maybe not since that seems more about the main parser and not the subparsers..  not sure, carry on.